src/azure-cli/azure/cli/command_modules/batch/tests/latest/test_batch_mgmt_commands.py

```diff
@@ -343,6 +343,10 @@ def test_batch_application_cmd(self, resource_group, storage_account):
 
 
 class BatchMgmtByosScenarioTests(ScenarioTest):
+
+    def __init__(self, method_name):
+        super().__init__(method_name, random_config_dir=True)
```

Comment:
`*arg` and `*kwargs` should pass through. See #25689.

Reply:
Updated